riotfile.py

```diff
@@ -2604,7 +2604,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={"requests": latest, "gevent": latest},
             venvs=[
                 Venv(
-                    pys=select_pys(min_version="3.8"),
+                    pys=select_pys(),
```

Comment:
I think we might need to keep this constraint until #5105 is merged, but let's see what the CI says.

Reply:
Yup, just added this temporarily to verify which versions work under this change 😄